cmd/operator-sdk/add/api.go

```diff
@@ -135,24 +97,6 @@ func apiRun(cmd *cobra.Command, args []string) error {
 		if err := doAnsibleAPIScaffold(); err != nil {
 			return err
 		}
-	case projutil.OperatorTypeHelm:
-		absProjectPath := projutil.MustGetwd()
-		projectName := filepath.Base(absProjectPath)
-		cfg := input.Config{
-			AbsProjectPath: absProjectPath,
-			ProjectName:    projectName,
-		}
-		createOpts := helm.CreateChartOptions{
-			ResourceAPIVersion: apiFlags.APIVersion,
-			ResourceKind:       apiFlags.Kind,
-			Chart:              apiFlags.HelmChartRef,
-			Version:            apiFlags.HelmChartVersion,
-			Repo:               apiFlags.HelmChartRepo,
-			CRDVersion:         apiFlags.CrdVersion,
-		}
-		if err := helm.API(cfg, createOpts); err != nil {
-			return err
-		}
```

Comment:
Nit: Maybe leave this like with the Go operator:
```suggestion
       case projutil.OperatorTypeHelm:
               return fmt.Errorf("the `add api` command is not supported for Helm operators")